config/api/README.md

```diff
@@ -0,0 +1,26 @@
+# API Configuration
```

Comment:
If this is an API configuration file, we wouldn't need the portfolio* properties, could you please clean up the read me file here under api?